include/dem/output_force_torque_calculation.h

```diff
@@ -43,14 +44,12 @@ write_forces_torques_output_locally(
 template <int dim>
 void
 write_forces_torques_output_results(
-  const std::string               filename,
-  const unsigned int              output_frequency,
-  const std::vector<unsigned int> boundary_index,
-  const double                    time_step,
-  std::map<unsigned int, std::map<unsigned int, dealii::Tensor<1, 3>>>
-    &forces_boundary_information,
-  std::map<unsigned int, std::map<unsigned int, dealii::Tensor<1, 3>>>
-    &torques_boundary_information)
+  const std::string                                filename,
+  const unsigned int                               output_frequency,
+  const std::vector<unsigned int>                  boundary_index,
+  const double                                     time_step,
+  DEM::dem_data_structures<3>::vector_on_boundary &forces_boundary_information,
+  DEM::dem_data_structures<3>::vector_on_boundary &torques_boundary_information)
```

Comment:
vector_on_boundary is `std::unordered_map<unsigned int, std::map<types::boundary_id, Tensor<1, 3>>>`
From the following line, the key of the 2nd map [i] is the # of processor and not a boundary id. Unsigned int is right, but not the typedef.

Reply:
That's weird, it means this "bug" was kinda always there because in the past the vector_on_boundary was being cast into this. I'll look more into this and get back to you.